pulsar-client-cpp/tests/BasicEndToEndTest.cc

```diff
@@ -2327,7 +2327,8 @@ TEST(BasicEndToEndTest, testSyncFlushBatchMessagesPartitionedTopic) {
     consConfig.setConsumerType(ConsumerExclusive);
     consConfig.setReceiverQueueSize(2);
     ASSERT_FALSE(consConfig.hasMessageListener());
-    Consumer consumer[numberOfPartitions];
+    std::vector<Consumer> consumer(numberOfPartitions);
+    // Consumer consumer[numberOfPartitions];
```

Comment:
Can you remove the commented lines if they are not needed?

Reply:
OK, I will.